app/controllers/news_controller.rb

```diff
@@ -5,7 +5,7 @@ class NewsController < ApplicationController
 
 
   def index
-    @articles = NewsArticle.published.by_release_date.by_name
+    @articles = NewsArticle.published.by_release_date.by_published_at.by_name
```

Comment:
Do we need it to be alphabetical? I fully acknowledge now it's next to impossible to be sorted by alphabetical order when it's to the timestamp.